prisma/migrations/20211225043237_add_transaction_table/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- CreateTable
+CREATE TABLE "Transaction" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL,
+    "userId" TEXT NOT NULL,
+    "subscriptionId" TEXT NOT NULL,
+    "bankName" TEXT NOT NULL,
+    "bankAccountNumber" TEXT NOT NULL,
+    "bankAccountName" TEXT NOT NULL,
+    "amount" INTEGER NOT NULL,
```

Comment:
Mas @zainfathoni Is the price always fixed? If yes using integer are sufficient enough, if not I prefer string/money type. Because of floating point, in js. And then use [currency.js](https://currency.js.org/) or [dinero](https://dinerojs.com/) to deal with money thing.

Reply:
According to this [Transaction Flow](https://github.com/zainfathoni/kelas.rumahberbagi.com/blob/main/docs/diagrams.md#transaction-flow) diagram, we are only performing the **read** operation on the amount, which is for the **Author** to verify the transfer.

We won't be doing any numerical operations anytime soon. Therefore, we can keep this suggestion for future improvements. It's a good idea, though. If you don't mind, would you please document this proposal as a _feature request_ issue in this repository, @mandaputtra? 😁 Thanks! 🙏